controllers/storagecluster/storageclient.go

```diff
@@ -0,0 +1,160 @@
+package storagecluster
+
+import (
+	"crypto/tls"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"time"
+
+	ocsclientv1a1 "github.com/red-hat-storage/ocs-client-operator/api/v1alpha1"
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v4/v1"
+	ocsinit "github.com/red-hat-storage/ocs-operator/v4/controllers/ocsinitialization"
+	extv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	cutil "sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+const (
+	storageClientCRDName   = "storageclients.ocs.openshift.io"
+	storageClientName      = "local-storageclient"
+	onboardingTokenURLPath = "onboarding-tokens"
+	saTokenPath            = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+)
+
+type storageClient struct{}
+
+var _ resourceManager = &storageClient{}
+
+func (s *storageClient) ensureCreated(r *StorageClusterReconciler, storagecluster *ocsv1.StorageCluster) (reconcile.Result, error) {
```

Comment:
It is not being called, Can you pls add this struct in the `reconcile.go.`

Reply:
>  Can you pls add this struct
- done.

Reply:
@iamniting Resolving this as I verified the change was done